docker/build.sh

```diff
@@ -214,6 +206,10 @@ function docker_cmd()
             $DOCKER_REMOTES_BINARY tag $IMAGE:${DOCKER_TAG} $IMAGE:${GIT_BRANCH}
             $DOCKER_REMOTES_BINARY push $IMAGE:${GIT_BRANCH}
 
+            echo "pushing $GCR_IMAGE docker image..."
```

Comment:
Would it make sense to support either dockerhub, GCR, or both? I could see some users wanting to build Leo and only push to 1 place. Maybe could have some logic like:
```
if [ -n $IMAGE ]; then
  # push to dockerhub
fi
if [ -n $GCR_IMAGE ]; then
  # push to GCR
fi
```
This would mean not having defaults for the dockerhub/GCR registries which may make sense as we specify them in `jenkins_build.sh` anyway.

Reply:
i was wondering that as i was making this change.  I can set up that functionality, i agree that it'd be useful.